addons/armaos/CfgHackingCommands.hpp

```diff
@@ -0,0 +1,20 @@
+class HackingCommand
```

Comment:
Is it possible, that all commands are inherited by the same parent (OsFunction)?

Reply:
I asked myself the same question. Because I need to filter for specific classes, it was easier to introduce completely new classes. But I could do something like inheriting from `OsFunction` or I could add an additional value/parameter for filter purposes only. I need to think about that.

Reply:
Will do that.